Gems/Atom/RHI/Vulkan/Code/Source/RHI/TransientAttachmentPool.cpp

```diff
@@ -144,7 +144,7 @@ namespace AZ
             if (descriptor.m_renderTargetBudgetInBytes || allowNoBudget)
             {
                 // Use an image descriptor of size 1x1 to get the memory requirements.
-                RHI::ImageBindFlags bindFlags = RHI::ImageBindFlags::Color | RHI::ImageBindFlags::DepthStencil | RHI::ImageBindFlags::ShaderRead;
+                RHI::ImageBindFlags bindFlags = RHI::ImageBindFlags::Color | RHI::ImageBindFlags::ShaderRead;
```

Comment:
Why remove DepthStencil?

Reply:
The vk GetImageMemoryRequirements function won't work with both color and depthstencil active.
Basically you can't create a image with both flags active at same time. and Vulkan would report an error/warning.

Reply:
I think what that means is that the driver is saying that depthstencil and color has conflicting mem requirements so pick one or the other. With this change we are basically ignoring DepthStencil. I wonder if we need a different pool for DepthStencil RT vs Color. I am assuming that this works so I guess we can accept it for now.

Reply:
Based on our conversation this is fine